terraform_shared/secretsmanager.tf

```
@@ -2,4 +2,8 @@ resource "aws_secretsmanager_secret" "notify_api_key" {
   name = "notify-api-key"
 
   tags   = "${local.default_tags}"
+}
+
```

because this has moved out of shared, you need another data source to pull in the secret id you're using.
So,

Get the secret id 
use secret id to get secret value


You can insert this above aws_secretsmanager_secret_version
```suggestion
data "aws_secretsmanager_secret" "notify_api_key" {
  name = "notify-api-key"
}

```